### PR TITLE
Adaptive Card v1.6

### PR DESCRIPTION
**Pull request by sowrabh-msft**
## Changes made:
- Add support for top level property `metadata` and `metadata.webUrl`
- Add support for `expires` property in `refresh`

### Metadata and `metadata.webUrl`
Metadata is a top level property to host non-functionality related metadata properties that do not affect rendering or actionability. The first property we are introducing here is the `webUrl` which can be used as a fallback by hosts to redirect users to the URL and uniquely map cards to their source URL.

### `refresh.expires` property
`refresh.expires` is a way for card authors to indicate the freshness of content delivered. The canonical scenario is a flight tracking card. If the flight is a month away, the developer could set the expiration weekly, but as the flight time approaches the expiration window can be reduced. 

The client is able to use this information as it sees fit, up to and including automatically refreshing the content if the expiration time has passed or avoiding refresh while the content is still valid.

### Example payload
```json
{
    "type": "AdaptiveCard",
    "version": "1.0",
    "metadata": { 
        "webUrl": "[https://](https://../)asana.com/ticket/5"
    },
    "refresh": {
        "expires": "2022-01-01T12:00:00Z",
        "action": { 
            "type": "Action.Execute",
            "verb": "..."   
         }
    },
    "body": { ... }
}
```

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/7105)

### CODE REVIEW

File: schemas/src/shared/Refresh.json [hunk 1]
```diff
@@ -7,6 +7,10 @@
 			"type": "Action.Execute",
 			"description": "The action to be executed to refresh the card. Clients can run this refresh action automatically or can provide an affordance for users to trigger it manually."
 		},
+		"expires": {
+			"type": "string",
+			"description": "Hint to host that a refresh to card wouldn't be needed till timestamp. The timestamp is expected to be in ISO-8601 Instant format.  E.g. 2022-01-01T12:00:00Z"
+		},
```
Comment (matthidinger) [inline schemas/src/shared/Refresh.json:13]: Suggested description: "A timestamp that informs a Host when the card content has expired, and that it should trigger a refresh as appropriate. The format is ISO-8601 Instant format. E.g., 2022-01-01T12:00:00Z"

Comment (karthikbaskar-ms) [reply schemas/src/shared/Refresh.json:13]: Updated as suggested

**Review by paulcam206 — APPROVED** [review]
looks good -- should fix indentation in `AdaptiveCard.json` though...

File: schemas/src/AdaptiveCard.json [hunk 2]
```diff
@@ -50,6 +50,11 @@
 				}
 			]
 		},
+		"metadata": {
+			"type": "Metadata",
+			"description": "Defines various metadata properties typically not used for rendering the card",
+			"version": "1.6"
+        },
```
[the remark this answers is not in the document]
Comment (karthikbaskar-ms) [reply schemas/src/AdaptiveCard.json]: Fixed the indentation

File: schemas/src/shared/Refresh.json [hunk 3]
```diff
@@ -7,6 +7,10 @@
 			"type": "Action.Execute",
 			"description": "The action to be executed to refresh the card. Clients can run this refresh action automatically or can provide an affordance for users to trigger it manually."
 		},
+		"expires": {
+			"type": "string",
+			"description": "A timestamp that informs a Host when the card content has expired, and that it should trigger a refresh as appropriate. The format is ISO-8601 Instant format. E.g., 2022-01-01T12:00:00Z"
```
Comment (matthidinger) [inline schemas/src/shared/Refresh.json:12]: I think version: 1.6 is missing from here.

**Review by matthidinger — CHANGES_REQUESTED** [review]

**Review by matthidinger — APPROVED** [review]

**Review by matthidinger — APPROVED** [review]

**Pull request merged**